### ISSUE TITLE
Resource Instance Widgets - 'Edit', 'Delete' and 'Info' buttons are missing Aria labels

### ISSUE THREAD

**Issue by rlwilliamss** [archesproject/arches#8955]
**Describe the bug**
Accessibility guideline 1.1.1 Non-Text Content.
Resource Instance Widgets - 'Edit', 'Delete' and 'Info' buttons are missing Aria labels.

**To Reproduce**
1. Navigate to Consultation workflow
2. Add a Related Application Area using form control
3. Enable NVDA screen reader
4. Using keyboard, move to the newly created relationship reference and listen to interpretation of 'Edit', 'Delete' and 'Info' buttons.
5. Click on Reference name to open reference details
6. Enable WAVE browser extension and view empty button errors - the icons in each reference are all read out as 'Blank' by the screen reader

**Screenshots**
![image](https://user-images.githubusercontent.com/78143228/188861447-e6c887b1-aea4-46be-9b30-8d993665539c.png)

![image](https://user-images.githubusercontent.com/78143228/188861624-f42e5351-9645-449c-9244-6a9b73776fac.png)

![image](https://user-images.githubusercontent.com/78143228/188861655-3c40e780-fde5-4227-8ea9-07b0647bde8b.png)

![image](https://user-images.githubusercontent.com/78143228/188861690-4410fca8-7783-48c3-99ff-958a028416d9.png)

**Expected behavior**
All non-text content should have associated aria labels to ensure appropriate interpretation by screen readers.

**Your Arches Information**
Version used: 6.1
Operating System and version (desktop or mobile): Windows 10
Browser Name and version: Chrome 102.0.4951.67, Firefox 91.9.0esr, Edge Version 101.0.1210.47, Safari 12.1
Link to your Arches Install (optional):

**Ticket Background**
*   Found by:
